pkgs/development/libraries/glog/default.nix

```diff
@@ -28,7 +28,9 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
-  cmakeFlags = [ "-DBUILD_SHARED_LIBS=ON" ];
+  propagatedBuildInputs = [ gflags ];
+
+  cmakeFlags = [ "-DBUILD_SHARED_LIBS=${if static then "OFF" else "ON"}" ];
```

Comment:
This merge is not correct, the static option was removed in [4541012](https://github.com/NixOS/nixpkgs/pull/77047/commits/4541012bdbcb5da522e056b75808fa5fa263fc23) which is part of this PR. `cmakeFlags` can be changed back and the `static` option can be removed.

Reply:
Please open a new PR with the fix.

Reply:
#77682